### PR TITLE
node upgrade, working docker dev setup

### PR DESCRIPTION
**Pull request by solomon-negusse**

### CODE REVIEW

File: docker-compose-develop.yml [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
-  test:
+  develop:
     build: .
+    ports:
+      - "30518:30518"
     command: develop
     environment:
       MONGO_DATABASE: 'doc-importer-tasks'
-      CT_REGISTER_MODE: auto
+      MONGO_PORT_27017_TCP_ADDR: mymachine
```
Comment (tiagojsag) [inline docker-compose-develop.yml]: This is out of scope, and a bad solution to the problem you're trying to solve, as it won't work without custom hosts.

File: Dockerfile [hunk 2]
```diff
@@ -24,7 +24,7 @@ WORKDIR /opt/$NAME
 COPY ./app /opt/$NAME/app
 RUN chown $USER:$USER /opt/$NAME
 
-EXPOSE 5000
+EXPOSE 30518
```
Comment (tiagojsag) [inline Dockerfile]: Please see the related comment on a different PR.

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: docker-compose-develop.yml (hunk 1 above)
Comment (solomon-negusse) [reply docker-compose-develop.yml]: Indeed, updated.

File: Dockerfile (hunk 2 above)
Comment (tiagojsag) [reply Dockerfile]: this still needs to be reverted

File: docker-compose-develop.yml [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
     environment:
       MONGO_DATABASE: 'doc-importer-tasks'
-      CT_REGISTER_MODE: auto
+      MONGO_PORT_27017_TCP_ADDR: mongo
+      MONGO_PORT_27017_TCP_PORT: 27017
+      RABBITMQ_URL: amqp://mymachine:5672
+      CT_REGISTER_MODE: manual
       CT_URL: http://mymachine:9000
-      LOCAL_URL: http://mymachine:5000
+      ELASTIC_URL: https://mymachine:9200
```
Comment (tiagojsag) [inline docker-compose-develop.yml:15]: you still have "mymachine" references

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (solomon-negusse) [reply Dockerfile]: thanks, the change wasn't committed yet.
